migration/20211109-update-licenses.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE licenses RENAME COLUMN remaining_checkouts TO checkouts_left;
+ALTER TABLE licenses RENAME COLUMN concurrent_checkouts TO checkouts_available;
+CREATE TYPE licensestatus AS ENUM ('preorder', 'available', 'unavailable');
+ALTER TABLE licenses ADD COLUMN status licensestatus;
+UPDATE licenses SET status = 'available';
+ALTER TABLE licenses ADD COLUMN terms_concurrency INTEGER;
```

Comment:
Since `License` is only used by ODL at the moment, I renamed the columns to reflect their names in OPDS2+ODL, instead of the names from OPDS+ODL we were using before. I think we will have more usage of OPDS2+ODL going forward.

Reply:
I had to do a database migration anyway to add the `status` field to licenses, so it seemed like a good time to change the names as well.

Reply:
Running the migration, I ran into
```
psycopg2.errors.DuplicateObject: type "licensestatus" already exists
```
I'm going to pull a recent database to ensure that it was just in my environment, which might've been tainted somehow.

If it's a more widespread issue, then might want to handle the `duplicate_object` exception.

I'll update this thread once I've checked.

Reply:
I saw this as well. Updated to use the SQL you suggested that handles the exception gracefully.